app/src/main/java/org/schabi/newpipe/player/gesture/PopupPlayerGestureListener.kt

```diff
@@ -251,8 +249,7 @@ class PopupPlayerGestureListener(
         // -- Determine if the ClosingOverlayView (red X) has to be shown or hidden --
         val showClosingOverlayView: Boolean = playerUi.isInsideClosingRadius(movingEvent)
         // Check if an view is in expected state and if not animate it into the correct state
-        val expectedVisibility = if (showClosingOverlayView) View.VISIBLE else View.GONE
-        if (binding.closingOverlay.visibility != expectedVisibility) {
+        if (binding.closingOverlay.isGone) {
```

Comment:
I think this is what you meant
```suggestion
        if (binding.closingOverlay.isVisible != showClosingOverlayView) {
```

Reply:
oh my god

![7sj2qs](https://github.com/TeamNewPipe/NewPipe/assets/32376686/2b4e6d32-a4dd-41f0-a961-faf6a6c51af9)